App.js

```diff
@@ -1,10 +1,15 @@
 import React from 'react';
-import {StyleSheet, View} from 'react-native';
-import {Text, Icon} from 'react-native-elements';
-import TodoList from './src/components/TodoList';
+import {StyleSheet, View, Text} from 'react-native';
+import {Icon} from 'react-native-elements';
+import TodoList from './src/components/TodoList/TodoList';
+import { Font } from 'expo';
+import TodayText from './src/components/TodayText/TodayText'
```

Comment:
and...this is creating a custom Text component that will use our font, because we'll either have to add the font to the style of every single Text component we're going to use, or we can just create a custom Text component.  This seemed easier for future dev.  We can rename if TodayText is a bad name. (it is)